### DISCUSSION TITLE
美西目标域名推荐

### DISCUSSION THREAD

**Discussion by TingTing1990** [XTLS/Xray-core#2256] [Q&A]
新手，请问下美西有什么目标域名推荐？
或者是用自己网站域名更好？

**Comment by vonl1** [comment]
### yahoo.com
再不行`microsoft.com`

**Comment by RPRX** [comment]
有自己的域名就用自己的域名，没有的话建议用 https://github.com/XTLS/RealiTLScanner 扫邻居的，除非你在白名单地区不得不用白名单域名

https://github.com/XTLS/Xray-core/issues/2247#issuecomment-1606004024 微软、苹果用的人太多，已不建议；有些域名其实在国内有 CDN，也不建议（听说微软、谷歌就是这样）

@chika0801 经常推荐用当地旅游相关、大学官网之类的域名，我觉得挺好，且这些域名可能也在白名单内

要注意最低标准、加分项 https://github.com/XTLS/REALITY ，以及 `dest` 的选择对 REALITY 表现的影响 https://github.com/XTLS/Xray-core/issues/2017#issuecomment-1532345891 https://github.com/XTLS/Xray-core/issues/2017#issuecomment-1532359978

**Reply by ssmetall** [reply]
Hello rprx, currently only one sni is working in one of Iran's mobile operators and it is expanding to local servers and soon they will all be blacklisted. If this is also blocked, we Iranians will not have internet access We will not have it, please give us a solution

**Reply by ssmetall** [reply]
almost 10 thousand sni's I checked, unfortunately only one of them works, please help us

**Reply by chika0801** [reply]
Your country's blockade is a hellish level of difficulty, and we can only try to help, but trying our best does not mean that we are obliged, necessarily, to respond to you in a timely manner, so please understand.

**Reply by hawshemi** [reply]
@chika0801 is it more complex than China?

**Reply by EbrahimTahernejad** [reply]
> @chika0801 is it more complex than China?

At some levels, they act similarly. It's probably just that the mullahs set it on "I'm scared I'm gonna lose my throne" mode.

And yes maybe someday China would do the same.

BTW I really appreciate what you guys are doing here.

**Reply by toyo2333** [reply]
我找了个当地旅游局的域名 符合基础条件，然后去查了下域名的ip是cloudflare的 这种能行么？

**Reply by chika0801** [reply]
能行的

**Reply by TingTing1990** [reply]
多谢！

**Reply by leslieqwq** [reply]
可以偷自己在国内云服务商买的并且备案过的域名吗？

**Reply by aptexd** [reply]
> 可以偷自己在国内云服务商买的并且备案过的域名吗？

我倒是找到一个可用的，同IP段的cn域名，哈哈哈哈

**Comment by chika0801** [comment]
事件记录：

7月14日晚上约11点-12点期间，在Xray群陆续有群友说REALITY用的微软的域名，客户端不通。观察群消息，用XUI默认DEST填的是微软的这样中招的推测最多，印象中应该有一键脚本也是DEST默认用微软的。来反馈的群友换了其它DEST后，客户端就通了的占大多数。

也有不少2位说换了DEST为其它网址还是不通，只能给他们说排查VPS本身端口IP回程线路正常不。REALITY相关在以前通，现在突然不通时，排查下DEST填其它网址，自己来测试。

又有1位回帖说是微软网炸了

**Reply by 2275273** [reply]
我大概14号下午3点左右reality就全挂了。。

**Reply by lanvada** [reply]
不只是微软域名出了问题吧？
我三个使用的reality的服务器，全挂了。在aws上购买的，分别在法兰克福，大阪和巴黎
另外还有三个使用vision的也挂了，这个是用我自己的网站伪装的，两个在东京一个在新加坡，也是aws购买的。
但是我已经在床上了。。。我打算明天再检查咋回事
之前一直稳定运行了一年左右，是不是最近又严打了啊。。。

**Reply by chika0801** [reply]
https://github.com/XTLS/Xray-core/issues/2251#issuecomment-1636025258

昨天有发英文的人在回帖，然后还有1人跟帖

**Reply by chika0801** [reply]
@lanvada 我现在，看了下我的一些偷自己，和非偷自己的VPS，都正常

**Reply by chika0801** [reply]
https://t.me/projectXray/2582199

> 我也不知道，之前用微软的，都好好的，今天不能用，那我就换三星的，还是不行😭😭😭，有没有人用xui改成功过的

我观察了下用XUI遇到的，我推测占反馈群友中的多数。

**Reply by chika0801** [reply]
![photo_2023-07-15_01-05-06](https://github.com/XTLS/Xray-core/assets/88967758/73d89864-1bb6-4e51-bb01-ea6460d58896)

https://t.me/projectXray/2582206

破案了？

**Reply by chika0801** [reply]
一个联动 https://github.com/XTLS/Xray-core/issues/2251#issuecomment-1636150465

**Reply by sjy5270882** [reply]
能说一下怎么偷自己吗，想学个教程，有大佬分享吗

**Reply by ccckfg** [reply]
我偷的某个微软域名还没炸٩(๑ ͡° ͜ʖ ͡°๑)۶

**Reply by chika0801** [reply]
@sjy5270882 https://github.com/chika0801/Xray-examples/tree/main/VLESS-XTLS-uTLS-REALITY/steal_yourself 这是示例，没详细文章，自己看看了

**Reply by lanvada** [reply]
感谢大佬，确实应该是Microsoft的域名有些问题，我今天上班换成别的试试

**Reply by lanvada** [reply]
> @sjy5270882 https://github.com/chika0801/Xray-examples/tree/main/VLESS-XTLS-uTLS-REALITY/steal_yourself 这是示例，没详细文章，自己看看了

大佬，想问一下偷自己是指流量进入自己搭的域名和服务器吗？我看这个示例里面似乎是用nginx代理到了其他的地方？这种算是偷自己嘛

**Reply by chika0801** [reply]
操作上来说dest那填的是本机一个端口号 127.0.0.1 被省略了，你写上也没问题。nginx监听本机这个端口，ng加载SSL证书，你把ng换其它caddy也可以

**Reply by lanvada** [reply]
> 操作上来说dest那填的是本机一个端口号 127.0.0.1 被省略了，你写上也没问题。nginx监听本机这个端口，ng加载SSL证书，你把ng换其它caddy也可以

好的👌蟹蟹大佬。另外想问一下dest和ServerName，会导致延迟上升和网速下降吗？我按照上面那个示例修改了config，但是我访问Youtube的延迟和网速的表现似乎都变差了，但是访问测速的网站倒是还正常。尤其是网速和使用vision的服务器相比有明显下降，我不知道问题出在了哪里。。。

**Reply by chika0801** [reply]
我在dest那用偷自己形式，还是用别人的网址的形式，我都没感觉到基于同一VPS不变时，客户端浏览网页主观感觉有不有变慢。你为什么遇到，我也说不清楚。网上自建就是一个折腾自学，问人测试的道路。享受这折腾的过程吧。。。

**Reply by chika0801** [reply]
能建议你的是，你不要只拿1个VPS测试，假设你现在这个VPS遇到了，你可以找其它VPS复测。如果复测了1-2个VPS都有。你再来发帖子描述问题求助。

**Reply by chika0801** [reply]
别人没遇到，我回你没遇到最后只有你能排查你自己的VPS，自己对比测试，是一个折腾的过程。

**Reply by lanvada** [reply]
ok 我再测试测试

**Reply by lanvada** [reply]
> 别人没遇到，我回你没遇到最后只有你能排查你自己的VPS，自己对比测试，是一个折腾的过程。

#2366 我上周修好的所有服务器又同时崩了。。。域名和IP还能访问但是V2rayN和SSR都连不上。。。哭了

**Reply by chika0801** [reply]
帮不了你呀，一是你的情况只有你自己知道，每个人网络知识水平不同，排查能力也不一样，大约网上交流时，别人凭他经验说了一些可能性。剩下你自己的VPS，你自己搭的，出了状态，也只有你自己排查。自建就是这样一个学运维，折腾的道理了。当然你遇到死路时，需要别人帮助，也能随时问，不过能不能遇到正好有热心人看到自己的求助，也是要运气吧。（都这样自学过来的）

**Reply by lanvada** [reply]
好吧，我自己再研究研究，非常感谢！

**Reply by chika0801** [reply]
如果需求及时类求助，去xray的tg群，当然能不能遇到会的人，也是看自己运气缘分了。
